packages/core/src/shared/logger/logger.ts

```diff
@@ -105,11 +105,6 @@ const logLevels = new Map<LogLevel, number>([
 export type LogLevel = 'error' | 'warn' | 'info' | 'verbose' | 'debug'
 
 export function fromVscodeLogLevel(logLevel: vscode.LogLevel): LogLevel {
-    if (!vscode.LogLevel) {
```

Comment:
minimum version has been bumped since this was written https://github.com/aws/aws-toolkit-vscode/blob/a7732b96982adbbac691f2869db3136c159d80ed/packages/core/package.json#L8